source/_includes/asides/component_navigation.html

```diff
@@ -37,6 +37,16 @@
     {%- endif -%}.
   </div>
 
+  {% if page.works_with %}
```

Comment:
Let's adjust this check to ensure it has a codeowner set and meets the integration quality scale requirement.